src/librawspeed/tiff/TiffEntry.h

```diff
@@ -78,7 +78,7 @@ class TiffEntry
   ushort16 getShort(uint32 num=0) const;
   short16 getSShort(uint32 num=0) const;
   float getFloat(uint32 num=0) const;
-  std::string getString() const;
+  std::string getString(bool trimmed = true) const;
```

Comment:
> In any case, getId() needs to keep calling trim.

Ok, so I'll add that back. Still: `getString(bool b=true)` or `getTrimmedString()`?